core-test/src/main/java/org/openstack4j/api/barbican/SecretTests.java

```diff
@@ -47,14 +48,25 @@ public void testGetSecret() throws IOException {
         Secret secret = osv3().barbican().secrets().get(secretId);
         assertNotNull(secret);
         assertNotNull(secret.getName());
+        assertEquals(secret.getExpiration().toString(), expiration);
```

Comment:
Instead of comparing the strings, I'd suggest to compare the Date types. Make sense? I promise that's the last comment @rschollmeyer.

Reply:
Sure i can do that. Don't worry, i am thankful for every good critique!

Reply:
To be sure: do you mean to compare the date formats?

Reply:
Compare `assertEquals(secret.getExpiration(), expiration);` where expiration is a `Date` object